<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1718322582804' stop='1718322583839'>
    <name>prueba e2e rodrigo, acceder a evaluacion solamente loggeado</name>
    <title>prueba e2e rodrigo, acceder a evaluacion solamente loggeado</title>
    <test-cases>
        <test-case start='1718322582804' status='broken' stop='1718322583787'>
            <name>should redirect to external login page if not logged in</name>
            <title>should redirect to external login page if not logged in</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>`cy.visit()` failed trying to load:
                
                http://localhost:3000/
                
                We attempted to make an http request to this URL but the request failed without a response.
                
                We received this error at the network level:
                
                  > Error: connect ECONNREFUSED 127.0.0.1:3000
                
                Common situations why this would fail:
                  - you don't have internet access
                  - you forgot to run / boot your web server
                  - your web server isn't accessible
                  - you have weird network configuration settings on your computer</message>
                <stack-trace>CypressError: `cy.visit()` failed trying to load:
                
                http://localhost:3000/
                
                We attempted to make an http request to this URL but the request failed without a response.
                
                We received this error at the network level:
                
                  > Error: connect ECONNREFUSED 127.0.0.1:3000
                
                Common situations why this would fail:
                  - you don't have internet access
                  - you forgot to run / boot your web server
                  - your web server isn't accessible
                  - you have weird network configuration settings on your computer
                at __webpack_modules__&lt;/__WEBPACK_DEFAULT_EXPORT__/visit/visit/ (http://localhost:58341/__cypress/runner/cypress_runner.js:131491:74)
                at visitFailedByErr (http://localhost:58341/__cypress/runner/cypress_runner.js:130889:12)
                at __webpack_modules__&lt;/__WEBPACK_DEFAULT_EXPORT__/visit/visit (http://localhost:58341/__cypress/runner/cypress_runner.js:131490:27)
                at tryCatcher (http://localhost:58341/__cypress/runner/cypress_runner.js:1807:23)
                at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromiseFromHandler (http://localhost:58341/__cypress/runner/cypress_runner.js:1519:31)
                at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromise (http://localhost:58341/__cypress/runner/cypress_runner.js:1576:18)
                at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromise0 (http://localhost:58341/__cypress/runner/cypress_runner.js:1621:10)
                at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromises (http://localhost:58341/__cypress/runner/cypress_runner.js:1697:18)
                at _drainQueueStep (http://localhost:58341/__cypress/runner/cypress_runner.js:2407:12)
                at _drainQueue (http://localhost:58341/__cypress/runner/cypress_runner.js:2400:24)
                at __webpack_modules__&lt;/Async.prototype._drainQueues (http://localhost:58341/__cypress/runner/cypress_runner.js:2416:16)
                at __webpack_modules__&lt;/Async/this.drainQueues (http://localhost:58341/__cypress/runner/cypress_runner.js:2286:14)
                From Your Spec Code:
                    at  (webpack://lifecheck-front/./cypress/e2e/braintests/rodrigo.cy.ts:15:0)
                    at __webpack_modules__&lt;/setRunnable/runnable.fn (http://localhost:58341/__cypress/runner/cypress_runner.js:138291:43)
                    at callFn (http://localhost:58341/__cypress/runner/cypress_runner.js:156911:21)
                    at __webpack_modules__&lt;/Runnable.prototype.run (http://localhost:58341/__cypress/runner/cypress_runner.js:156898:13)
                    at __webpack_modules__&lt;/create/onRunnableRun (http://localhost:58341/__cypress/runner/cypress_runner.js:162822:30)
                    at finallyHandler (http://localhost:58341/__cypress/runner/cypress_runner.js:4072:23)
                    at tryCatcher (http://localhost:58341/__cypress/runner/cypress_runner.js:1807:23)
                    at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromiseFromHandler (http://localhost:58341/__cypress/runner/cypress_runner.js:1519:31)
                    at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromise (http://localhost:58341/__cypress/runner/cypress_runner.js:1576:18)
                    at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromise0 (http://localhost:58341/__cypress/runner/cypress_runner.js:1621:10)
                    at __webpack_modules__&lt;/module.exports/Promise.prototype._settlePromises (http://localhost:58341/__cypress/runner/cypress_runner.js:1701:18)
                    at _drainQueueStep (http://localhost:58341/__cypress/runner/cypress_runner.js:2407:12)
                    at _drainQueue (http://localhost:58341/__cypress/runner/cypress_runner.js:2400:24)
                    at __webpack_modules__&lt;/Async.prototype._drainQueues (http://localhost:58341/__cypress/runner/cypress_runner.js:2416:16)
                    at __webpack_modules__&lt;/Async/this.drainQueues (http://localhost:58341/__cypress/runner/cypress_runner.js:2286:14)
                
                
                From Node.js Internals:
                  Error: connect ECONNREFUSED 127.0.0.1:3000
                      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1495:16)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>